### PR TITLE
[1] refactor: remove account/Affilate and its deps

### PR DESCRIPTION
**Pull request by shoom3301**
Just removed unused code related to `Affilate` component

## To test
No need testing

### CODE REVIEW

**Comment by shoom3301** [comment]
@alfetopito oh, I didn't know about this decision.
I think, we can restore it from git anytime and don't violate YAGNI

**Comment by shoom3301** [comment]
But if it's a problem, I can easily discard this PR

**Comment by alfetopito** [comment]
Raised in [this thread](https://cowservices.slack.com/archives/C0361CDG8GP/p1686234864965309) to decide if we keep it or not

**Review by anxolin — APPROVED** [review]
Let's do whatever is decided here https://cowservices.slack.com/archives/C0361CDG8GP/p1686234864965309

**Comment by alfetopito** [comment]
Can you rebase this PR? Too many unrelated changes now

File: src/utils/deepEqual.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+// https://dmitripavlutin.com/how-to-compare-objects-in-javascript/
+export function deepEqual(object1: any, object2: typeof object1): boolean {
```
Comment (alfetopito) [inline src/utils/deepEqual.ts:2]: Should this be part of this PR?

**Review by alfetopito — APPROVED** [review]
Approved, except this PR which should be removing code is adding the `deepEqual` utils

File: src/utils/deepEqual.ts (hunk 1 above)
Comment (shoom3301) [reply src/utils/deepEqual.ts:2]: Damn, it's an artifact after merging. Thanks

**Pull request merged**